services/rewards/cmd/rest_run.go

```diff
@@ -49,32 +50,41 @@ func RestRun(command *cobra.Command, args []string) {
 	ctx = context.WithValue(ctx, appctx.ParametersTransitionCTXKey, viper.GetBool("transition"))
 	// parse default-monthly-choices and default-tip-choices
 
-	var monthlyChoices = []float64{}
+	var monthlyChoices []float64
 	if err := viper.UnmarshalKey("default-monthly-choices", &monthlyChoices); err != nil {
-		logger.Fatal().Err(err).Msg("failed to parse default-monthly-choices")
+		lg.Fatal().Err(err).Msg("failed to parse default-monthly-choices")
 	}
 	ctx = context.WithValue(ctx, appctx.DefaultMonthlyChoicesCTXKey, monthlyChoices)
 
-	var tipChoices = []float64{}
+	var tipChoices []float64
 	if err := viper.UnmarshalKey("default-tip-choices", &tipChoices); err != nil {
-		logger.Fatal().Err(err).Msg("failed to parse default-tip-choices")
+		lg.Fatal().Err(err).Msg("failed to parse default-tip-choices")
 	}
 	ctx = context.WithValue(ctx, appctx.DefaultTipChoicesCTXKey, tipChoices)
 
-	var acChoices = []float64{}
+	var acChoices []float64
 	if err := viper.UnmarshalKey("default-ac-choices", &acChoices); err != nil {
-		logger.Fatal().Err(err).Msg("failed to parse default-ac-choices")
+		lg.Fatal().Err(err).Msg("failed to parse default-ac-choices")
 	}
 	if len(acChoices) > 0 {
 		ctx = context.WithValue(ctx, appctx.DefaultACChoicesCTXKey, acChoices)
 	}
 
-	// setup the service now
-	s, err := rewards.InitService(ctx)
+	tosVersion, err := strconv.Atoi(os.Getenv("REWARDS_TOS_VERSION"))
 	if err != nil {
-		logger.Fatal().Err(err).Msg("failed to initalize rewards service")
+		lg.Fatal().Err(err).Msg("error retrieving rewards terms of service version")
+	}
+
+	cfg := rewards.Config{
+		TosVersion: tosVersion,
```

Comment:
nit: TOS is an acronym, therefore:

```suggestion
		TOSVersion: tosVersion,
```

Reply:
updated